ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py

```diff
@@ -2152,4 +2151,38 @@ def test_paddle_instance_floor_(
         method_flags=method_flags,
         frontend=frontend,
         on_device=on_device,
-    )
\ No newline at end of file
+    )
+
+
+# bincount
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="bincount",
+    dtypes_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("int"),
```

Comment:
Did you setup locally to run the test? It seems like `int` is not a valid argument here when I try to run it. If you need help, please refer to the [Running Tests Locally Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#running-tests-locally) first.

Reply:
@xoiga123 Hi, I updated the test again, but for some reason I am not able to run the frontend tests on both my local and docker instances. I have asked for help to resolve that on Discord. But if possible could you check the test as well? Thank you